package.json

```diff
@@ -27,6 +27,7 @@
     "clarity-js": "^0.1.29",
     "compare-versions": "^3.1.0",
     "material-ui": "0.17.4",
+    "query-string": "^6.0.0",
```

Comment:
>  "query-string": "^6.0.0", [](start = 2, length = 27)

let's remove this and our reference to it in app.tsx - we don't actually use that parse anywhere